### DISCUSSION TITLE
Buildbot not mentioned anywhere

### DISCUSSION THREAD

**Discussion by cricri-pingouin** [omf2097/openomf#424] [General]
Is there any reason why there is no link to the buildbot anywhere on the Github page?
That's where I'm getting my Windows builds from:
https://buildbot.openomf.org/artifacts/
There is a Linux build too.

**Comment by katajakasa** [comment]
It's kind of unofficial source. We are moving towards github actions (https://github.com/omf2097/openomf/actions) -- the idea is to build and make automatic releases to github releases page (nightly releases). That one will be mentioned when someone has the time to finish the actions integrations :)

**Comment by katajakasa** [comment]
There we go. Latest build is now available here: https://github.com/omf2097/openomf/releases/tag/latest
